skyproj/vendor/mpl_toolkit_functions.py

```diff
@@ -0,0 +1,360 @@
+# These are mpl_toolking GridHelperCurveLinear functions vendored from matplotlib 3.8.
```

Comment:
You should include the short form the mpl license here at the top of the file to make the lawyers happy.

Reply:
Great point!